pom.xml

```diff
@@ -796,9 +796,9 @@
         <configuration>
           <errorRemedy>failFast</errorRemedy>
           <!--
-                                         The list of allowed licenses. If you see the build failing due to "There are some forbidden licenses used, please
-                                        check your dependencies", verify the conditions of the license and add the reference to it here.
-                                      -->
+                                                   The list of allowed licenses. If you see the build failing due to "There are some forbidden licenses used, please
```

Comment:
nit: this extra whitespace seems to happen almost every release pr? do we know why this is happening?